microsoft/testsuites/hpc/infinibandsuite.py

```diff
@@ -31,6 +32,33 @@
     """,
 )
 class InfinibandSuite(TestSuite):
+    @TestCaseMetadata(
+        description="""
+        This test case will
+        1. List all available network interfaces
+        2. Check if InfiniBand cards are present
+        3. Ensure InfiniBand cards are named starting with "ib0"
+        """,
+        priority=2,
+        requirement=simple_requirement(supported_features=[Infiniband]),
+    )
+    def verify_ib_naming(self, log: Logger, node: Node) -> None:
+        infiniband = node.features[Infiniband]
+        ib_interfaces = infiniband.get_ib_interfaces()
```

Comment:
[nit] You can directly write `node.features[Infiniband].get_ib_interfaces()`. We do this only if the tool is invoked multiple times

Reply:
Resolved.